src/raftstore/store/snap.rs

```diff
@@ -557,44 +912,9 @@ mod test {
         // temporary file should not be count in snap size.
         assert_eq!(mgr.rl().get_total_snap_size(), exp_len * 2);
 
-        mgr.rl().get_snap_file(&key2, true).unwrap().delete();
+        mgr.rl().get_snapshot_for_reading(&key2).unwrap().delete();
         assert_eq!(mgr.rl().get_total_snap_size(), exp_len);
-        mgr.rl().get_snap_file(&key2, false).unwrap().delete();
-        assert_eq!(mgr.rl().get_total_snap_size(), 0);
-
-        let mut f4 = mgr.rl().get_snap_file(&key2, true).unwrap();
```

Comment:
I don't think they are the same. `f4` here is to ensure the written data won't increase `total_snap_size` when not saved yet.